### ISSUE TITLE
Larger tap radius for action icons

### ISSUE THREAD

**Issue by seanmpuckett** [metabolist/metatext#251]
**Is your feature request related to a problem? Please describe.**

tapping icons eg favourite or boots often misses the small hit box causing unwanted action

**Describe the solution you'd like*

make hit box around action icons significantly larger

**Describe alternatives you've considered**

I cannot make my fingers smaller or shake less

**Additional context**

this is an accessibility issue

**Comment by alexbbrown** [comment]
Please can you take and share a screenshot of the UI where the issue occurs, with the specific element highlighted?

the Boost and Favorite UI appear in a number of similar looking contexts that actually have different code.  

Thanks!

**Comment by seanmpuckett** [comment]
![EB03612F-65C3-4919-9E7C-32F17D432D6A](https://user-images.githubusercontent.com/227003/204107771-d20122a3-1a9b-496e-b591-e8d8209ff96b.jpeg)

**Comment by seanmpuckett** [comment]
Tbh any use of small icon should have a large hit box for shaky fingers so a sweep through the entire app will be wise
